Cargo.toml

```diff
@@ -10,14 +10,18 @@ license = "MIT OR Apache-2.0"
 readme = "README.md"
 edition = "2018"
 
+[features]
+no-chip-version-assertion = []
+
 [dependencies]
 byteorder = { version = "1.3.4", default-features = false }
-embedded-hal = "0.2.4"
+embedded-hal = "0.2"
 embedded-nal = "0.6.0"
-bit_field = "0.10.1"
+bit_field = "0.10"
 derive-try-from-primitive = "1"
 nb = "1.0.0"
 defmt = { version = "0.3", optional = true }
 
-[features]
-no-chip-version-assertion = []
+[dev-dependencies]
+embedded-hal-mock = "0.9"
+defmt = { version = "0.3" }
```

Comment:
... why do we need defmt as a dev-dep? When we're running tests on the host, there's no point in deferring formatting of errors.

Reply:
Removed. I don't remember what I was/wanted to use it for